tests/tests_wireless_wpa3_owe_nm.yml

```diff
@@ -0,0 +1,21 @@
+# SPDX-License-Identifier: BSD-3-Clause
+# This file was generated by ensure_provider_tests.py
+---
+# set network provider and gather facts
+- hosts: all
+  name: Run playbook 'playbooks/tests_wireless_wpa3_owe.yml' with nm as provider
+  tasks:
+    - name: Set network provider to 'nm'
+      set_fact:
+        network_provider: nm
+      tags:
+        - always
+
+
+# The test requires or should run with NetworkManager, therefore it cannot run
+# on RHEL/CentOS 6
+# OWE has not been exposed via D-Bus on RHEL 7(dist-tag) and Fedora 32. Failed in setting up mock wifi on RHEL 8
+- import_playbook: playbooks/tests_wireless_wpa3_owe.yml
+  when:
+    - ansible_distribution_major_version != '6'
+    - ansible_distribution_major_version > '7' and ansible_distribution == 'CentOS' or ansible_distribution_major_version > '32' and ansible_distribution == 'Fedora'
```

Comment:
IMO, this line is too long (more than 160 chars). Could you split it? Thanks!

Reply:
I splitted the line.